app_flutter/lib/task_box.dart

```diff
@@ -29,10 +30,26 @@ class TaskBox extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    if (task.status == 'In Progress') {
```

Comment:
I would pull these string values into constants

Reply:
Done. I added a TODO to convert the usage to enums when the backend starts sending out protos.